### PR TITLE
Submission of HW-0.

### PR DESCRIPTION
**Pull request by amholtz**
Updated time.

### CODE REVIEW

**Comment by amholtz** [comment]
Thanks.

**Comment by rudeboybert** [comment]
Testing some stuff, so allow me to reopen.

File: HW-0.Rmd [hunk 1]
```diff
@@ -9,6 +9,6 @@ Your tasks!
 
 * Change the name of the author above to yours
 * Change the date
-* Replace the NHL URL with the URL for the document you just posted to Rpubs **here**: [My RPubs Document](https://www.nhl.com/)
+* Replace the NHL URL with the URL for the document you just posted to Rpubs **here**: [http://rpubs.com/aholtz/hw0]
```
Comment (rudeboybert) [inline HW-0.Rmd:12]: We'll learn more markdown as we go, but the correct syntax to have a URL show is:
`[TEXT TO DISPLAY](http://www.whatever.com)`

So in your case you should've had
`[My RPubs Document](http://rpubs.com/aholtz/hw0)`

As I may have said yesterday, Rpubs is a good tool for quickly sharing R Code/Output on the web with collaborators.

**Comment by rudeboybert** [comment]
Closing again. Thanks!

**Pull request closed without merging**